src/ledger/ledger-rules/README.md

```diff
@@ -0,0 +1,4 @@
+# Ledger Rules
+This section details the ledger rules used in Cardano and, especially, some of the potential "foot guns" one could run into when implementing these rules. Those "foot guns" could be discrepencies between the specification and the ledger, some undefined behavior, or a bug that was fixed at some point, but is still relevant for syncing historical data.
+
+It is an ongoing work in progress, being updated periodically in paralel with the work happening to build Amaru.
```

Comment:
I'm not sure if this is a very useful introduction yet. It seems you intend to describe only the discrepancies between specification and the haskell implementation? Can you please add links to those here?

Have you considered replacing / extending `ledger/README.md`?